### ISSUE TITLE
Linux Support

### ISSUE THREAD

**Issue by JM480** [retrowars/retrowars#77]
Can you add support for Linux and Linux mobile devices like the [Pinephone Pro?](https://www.pine64.org/pinephonepro/)

**Comment by pserwylo** [comment]
Absolutely. In fact, the framework used is [libGDX](https://libgdx.com), which supports Linux, Mac, Windows, and iOS. It technically supports HTML as well, but that doesn't work when using Kotlin (a known issue). Indeed when doing development, I actually do so on my Linux machine and conduct most playtesting there, with only a cursory test on my Android device prior to releasing.

Having said that, there are a few things which would need to be done for this to happen properly, mostly around figuring out how to publish various binaries. Instead of posting a checklist on this issue, I'll instead link to the official docs so that as time permits, we can check off each different platform:

https://libgdx.com/wiki/deployment/deploying-your-application

Note that although the first step is to post to GitHub pages, it would be great to also post to itch.io or other distribution platforms to make it more accessible to those who wish to play.

**Comment by SecularSteve** [comment]
Here's a Workaround for you at the moment.

It's like AnBox, but a ton better:

https://github.com/waydroid/waydroid
